### DISCUSSION TITLE
How to call external API from own API

### DISCUSSION THREAD

**Discussion by thesheppard** [koajs/koa#1681] [Q&A] [answered]
I have an issue where I want to call an external API to verify a certain token coming from my frontend.

i.e 

```js
router.post('/api/verifyToken', async (ctx, next) => {
   router.post('https://external-site.com/verify')
})
```

How would I go about doing this with koa?

**Comment by daniloab** [comment, marked as answer]
Fetch the api and do the next step accordingly with the return of it
